pathod/pathoc.py

```diff
@@ -73,7 +73,7 @@ def __str__(self):
             parts.append("\tPubkey: %s bit %s" % (pk.bits(), t))
             s = certutils.SSLCert(i)
             if s.altnames:
-                parts.append("\tSANs: %s" % " ".join(s.altnames))
+                parts.append("\tSANs: %s" % b" ".join(s.altnames))
```

Comment:
Shouldn't the format string also be bytes?
Or what happens if you format `str` with `bytes` arguments?